ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/ethernet-interfaces.j2

```diff
@@ -137,11 +137,15 @@ interface {{ ethernet_interface }}
 {%         elif ethernet_interfaces[ethernet_interface].spanning_tree_portfast is arista.avd.defined('network') %}
    spanning-tree portfast network
 {%         endif %}
-{%         if ethernet_interfaces[ethernet_interface].spanning_tree_bpdufilter is arista.avd.defined(true) %}
-   spanning-tree bpdufilter enable
-{%         endif %}
 {%         if ethernet_interfaces[ethernet_interface].spanning_tree_bpduguard is arista.avd.defined(true) %}
    spanning-tree bpduguard enable
+{%         elif ethernet_interfaces[ethernet_interface].spanning_tree_bpduguard is arista.avd.defined(false) %}
+   spanning-tree bpduguard disable
```

Comment:
I understand the logic here, but this could be a very breaking change, since disabling these knobs on interface level, instead of just not configuring them, would no longer honor the global setting on eos. Imagine the global is set to enabled. someone had `false` on bpduguard on some interfaces (maybe by mistake or maybe just because they tested it and it made no difference). With this change it would disable bpduguard for their interfaces.
A good example is our molecule scenarios, where this change affects configuration on multiple scenarios without you changing the data.
I would prefer if we could change the field from boolean to string, and then for backwards compatibility still accept the boolean `true`.
Ex.
```yaml
spanning_tree_bpdufilter: < "enabled" | true | "disabled" >
```
@carlbuchmann

Reply:
If that is the case, then we need to change values for bpdufilter as well!